### ISSUE TITLE
Fix canonical URLs in early Akka 2.5 docs

### ISSUE THREAD

**Issue by ennru** [akka/akka.github.com#606]
Eg. https://doc.akka.io/docs/akka/2.5.3/scala/guide/actors-intro.html has 

```
<link rel="canonical" href="https://doc.akka.io/docs/akka/current/../scala/guide/actors-intro.html"/>
```

which with redirects doesn't point to a similar page:

```
https://doc.akka.io/docs/akka/current/?language=scala/guide/actors-intro.html
```

**Comment by ennru** [comment]
This affected versions 2.5.2 through 2.5.6 quite broadly, but all other 2.5.x and 2.6.0-M1 versions' for some `Source-and-Flow` operators such as `2.5.18/stream/operators/Source-or-Flow/apply.html`.

I haven't touched 2.6.0-M1.

To fix the others I applied the following search-and-replace lines.

```
find . -name "*.html" -type f -exec sed -i -e 's/href="https:\/\/doc\.akka\.io\/docs\/akka\/current\/\.\.\/\.\.\/\.\./href="https:\/\/doc\.akka\.io\/docs\/akka\/current/g' {} \;

find . -name "*.html" -type f -exec sed -i -e 's/href="https:\/\/doc\.akka\.io\/docs\/akka\/current\/\.\.\/\.\./href="https:\/\/doc\.akka\.io\/docs\/akka\/current/g' {} \;

find . -name "*.html" -type f -exec sed -i -e 's/href="https:\/\/doc\.akka\.io\/docs\/akka\/current\/\.\./href="https:\/\/doc\.akka\.io\/docs\/akka\/current/g' {} \;
```

**Comment by ennru** [comment]
@raboof Would be good to look into what causes this in the current docs.

**Comment by raboof** [comment]
@ennru Are you saying this problem persists in the current version of the docs? Do you have an example?

**Comment by raboof** [comment]
(I checked https://doc.akka.io/docs/akka/current/stream/operators/Source-or-Flow/filter.html and https://doc.akka.io/docs/akka/current/guide/actors-intro.html and both seem fine unless I'm missing something)

**Comment by ennru** [comment]
I though of these
```
akkarepo@ip-10-0-5-133:~/www/docs/akka/2.6.0-M1$ grep -r  "href=\"https://doc.akka.io/docs/akka/current/\.\." .
./stream/operators/Source-or-Flow/partition.html:<link rel="canonical" href="https://doc.akka.io/docs/akka/current/../../../stream/operators/Source-or-Flow/partition.html"/>
./stream/operators/Source-or-Flow/mergePreferred.html:<link rel="canonical" href="https://doc.akka.io/docs/akka/current/../../../stream/operators/Source-or-Flow/mergePreferred.html"/>
./stream/operators/Source-or-Flow/broadcast.html:<link rel="canonical" href="https://doc.akka.io/docs/akka/current/../../../stream/operators/Source-or-Flow/broadcast.html"/>
./stream/operators/Source-or-Flow/mergeLatest.html:<link rel="canonical" href="https://doc.akka.io/docs/akka/current/../../../stream/operators/Source-or-Flow/mergeLatest.html"/>
./stream/operators/Source-or-Flow/balance.html:<link rel="canonical" href="https://doc.akka.io/docs/akka/current/../../../stream/operators/Source-or-Flow/balance.html"/>
./stream/operators/Source-or-Flow/unzip.html:<link rel="canonical" href="https://doc.akka.io/docs/akka/current/../../../stream/operators/Source-or-Flow/unzip.html"/>
./stream/operators/Source-or-Flow/unzipWith.html:<link rel="canonical" href="https://doc.akka.io/docs/akka/current/../../../stream/operators/Source-or-Flow/unzipWith.html"/>
./stream/operators/Source-or-Flow/mergePrioritized.html:<link rel="canonical" href="https://doc.akka.io/docs/akka/current/../../../stream/operators/Source-or-Flow/mergePrioritized.html"/>
./stream/operators/Source-or-Flow/apply.html:<link rel="canonical" href="https://doc.akka.io/docs/akka/current/../../../stream/operators/Source-or-Flow/apply.html"/>
./common/io-layer.html:<link rel="canonical" href="https://doc.akka.io/docs/akka/current/../common/io-layer.html"/>```

**Comment by ennru** [comment]
With the new protecting in Paradox we will hopefully avoid these going forward.
I've fixed the deployed Akka 2.6 docs manually now.
